src/NewTwitchApi/Resources/StreamsApi.php

```diff
@@ -54,9 +70,6 @@ public function getStreams(string $bearer, array $userIds = [], array $usernames
         foreach ($gameIds as $gameId) {
             $queryParamsMap[] = ['key' => 'game_id', 'value' => $gameId];
         }
-        foreach ($communityIds as $communityId) {
-            $queryParamsMap[] = ['key' => 'community_id', 'value' => $communityId];
-        }
```

Comment:
What do we want to do about the `array $communityIds = []` parameter in the method definition?

Reply:
While the preference would be to pull it I don't want to break users code for something so simple. We should just continue allowing data to be passed in, we just won't use it. Thoughts?

Reply:
I agree, but I don't want this to be forgotten and left in. Please create an issue for removal. Once removed, we'll bump major versions since BC will be broken.